mmcv/cnn/bricks/registry.py

```diff
@@ -7,7 +7,10 @@
 UPSAMPLE_LAYERS = Registry('upsample layer')
 PLUGIN_LAYERS = Registry('plugin layer')
 
+DROPOUT_LAYERS = Registry('Drop out layers')
+
 POSITIONAL_ENCODING = Registry('Position encoding')
 ATTENTION = Registry('Attention')
+FEEDFORWARD_NETWORK = Registry('Feed-forward Network')
```

Comment:
The name should be revised. I may suggest the widely used `MLP` for your reference. `Network` sounds like a complete model rather than a plugin layer.

Reply:
How about use FFN here? MLP is also acceptable to me.

Reply:
it conflicts with the class FFN in `transformer.py`